import_specifications/schema/dts_manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "resources": {"type": "array"},
+        "instructions": {
+            "type": "object",
+            "properties": {
+                "protocol": {"type": "string"},
```

Comment:
Should this be a CV?

Reply:
CV? crepuscular vole? crab vortex? constant value?

This one could be a constant, yes.

Reply:
controlled vocab

Reply:
This particular case should be a constant to be valid. Fixed.

Reply:
:+1: